src/CommandingOfficers/Commander.java

```diff
@@ -25,9 +25,7 @@
 import Terrain.GameMap;
 import Terrain.Location;
 import Terrain.TerrainType;
-import UI.UIUtils;
 import UI.UIUtils.Faction;
-import UI.Art.SpriteArtist.SpriteLibrary;
```

Comment:
Cleaning imports is a good thing, but it causes merge conflicts more often than not.
The codebase is full of serialization warnings anyway, so it doesn't even bother me to leave them.
...okay, maybe it does bother me. But merge conflicts bother me more.

Reply:
Hm. I think this was changed automatically because I committed a temp change in this class.

This branch is probably getting merged after all the others, which I guess means I'll have to fix any conflicts that come up anyway.